bazelrc/.bazelrc

```diff
@@ -0,0 +1,37 @@
+# Full content from https://blog.aspect.dev/bazelrc-flags
+
```

Comment:
nit: also link to cli flag docs for bazel; noticed you didn't link in the blog post